data/scripts/actions/objects/memorial.lua

```diff
@@ -0,0 +1,66 @@
+local lastUpdated = 0
```

Comment:
I think it's better to maintain the filename as 'outfit_memorial'. Because. memorial, for what? the filenames must be self-explanatory.

Reply:
memorial because the name of the object is memorial and not outfit memorial..

Reply:
ok. but, why have you removed royal outfit too ?